binary_search_trees/array_to_bst.py

```diff
@@ -10,4 +10,62 @@ def arr_to_bst(arr):
         Balanced Binary Search Tree using the elements in the array.
         Return the root of the Binary Search Tree.
     """
-    pass
\ No newline at end of file
+
+    class BinarySearch_tree:
+        def __init__(self) :
+            self.root   = None  
+
+        def insert(self, value):
+#      check for duplicates
+            if  value ==  self.root.value:
+                return 
+    #      If root is empty
+            mid_point  = len(arr)//2
+            if self.root == None:
+                self.root =  TreeNode(arr[mid_point])
+            else:
+                self.insertNode(value, self.root)
+
+# recursive function
+        def insertNode(self, value, current_node):
+            if value < current_node.value:
+                if current_node.left == None:
+                     current_node.left == TreeNode(value)                    
+#                       else set left to current_node
+                else:
+                    self.insertNode(value, current_node.left) 
+#           else check the right side
+            else:
+                if current_node.right == None:
+                    current_node.right = TreeNode(value)
+                else: 
+                    self.insertNode(value, current_node.right) 
+           
+
```

Comment:
It looks like you wrote out the logic to insert a node into a binary search tree which is great, but you're missing the logic to actually convert the array to the binary search tree.